src/parser/sparqlParser/SparqlQleverVisitor.cpp

```diff
@@ -121,7 +121,15 @@ ExpressionPtr Visitor::processIriFunctionCall(
       checkNumArgs(1);
       return sparqlExpression::makeTanExpression(std::move(argList[0]));
     }
-  }
+  } else if (checkPrefix(SCHEMA_TYPE)) {
+    if (functionName == "integer") {
+      checkNumArgs(1);
+      return sparqlExpression::toIntExpression(std::move(argList[0]));
+    } else if (functionName == "double") {
```

Comment:
```suggestion
    } else if (functionName == "double" || functionName == "decimal") {
```
(I think I left a comment somewhere)